src/components/views/messages/MessageActionBar.tsx

```diff
@@ -382,9 +382,18 @@ export default class MessageActionBar extends React.PureComponent<IMessageAction
                     'mx_MessageActionBar_expandMessageButton': !this.props.isQuoteExpanded,
                     'mx_MessageActionBar_collapseMessageButton': this.props.isQuoteExpanded,
                 });
+                const tooltip = <div>
+                    <div className="mx_Tooltip_title">
+                        { this.props.isQuoteExpanded ? _t("Collapse quotes") : _t("Expand quotes") }
+                    </div>
+                    <div className="mx_Tooltip_sub">
+                        { _t("click") }
```

Comment:
The shift part should be in this string, given the string should be an exact english translation (where pluralisation is not involved) , see https://github.com/vector-im/element-web/blob/develop/docs/translating-dev.md

Reply:
Should probably also have spaces around the `+`

<img width="174" alt="image" src="https://user-images.githubusercontent.com/2403652/157269991-116fb674-6b11-4bca-a3d3-29c4d737a633.png">